.github/workflows/zombie-bite-common.yml

```
@@ -328,6 +332,7 @@ jobs:
           echo "::endgroup::"
 
       - name: run_polkadot_ecosystem_tests
+        id: run_polkadot_ecosystem_tests
```

```suggestion
        id: run_polkadot_ecosystem_tests
        timeout-minutes: 15
```
@rockbmb  lets use 15 mins as timeout for now